study.md

```diff
@@ -49,8 +49,21 @@ A kitchen has many items.
 -   An herbs and spices rack containing chives, lemon grass, and mint on one
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
+
+
 ```json
-<!-- your answer here -->
+
+{
+"kitchen:" {
+"cupboard:" ["bowls", "plates"],
+"electric stove:" ["small", "medium", "large" "extra large"],
+"herb rack:" ["chives", "lemongrass" "mint"],
+"spices rack:" ["cinnamon", "nutmeg:", "vanilla"],
```

Comment:
Suuuper close here, just don't want the colon inside the string of the key

`"kitchen":`
`"cupboard":`

See?